services/offset-earth/offset-earth-trees.service.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { metric } = require('../text-formatters')
+const { floorCount } = require('../color-formatters')
+const { BaseJsonService } = require('..')
+
+const apiSchema = Joi.object({
+  total: Joi.number()
```

Comment:
Will these values be whole numbers? If so we can use the `nonNegativeInteger` helper from validators

https://github.com/badges/shields/blob/master/services/validators.js#L15